coalib/processes/BearRunning.py

```diff
@@ -160,9 +160,9 @@ def get_local_dependency_results(local_result_list, bear_instance):
 def run_local_bear(message_queue,
                    timeout,
                    local_result_list,
-                   file_dict,
```

Comment:
But it doesn't affect backwards compatibility in any way. It's just not passing the full dictionary around when we need only one element of it.